urllib3/poolmanager.py

```diff
@@ -110,8 +119,14 @@ class PoolManager(RequestMethods):
 
     proxy = None
 
-    def __init__(self, num_pools=10, headers=None, **connection_pool_kw):
+    def __init__(self, num_pools=10, headers=None, transport_security_manager=_DEFAULT_TSM,
+                 **connection_pool_kw):
         RequestMethods.__init__(self, headers)
+
+        if transport_security_manager is _DEFAULT_TSM:
+            transport_security_manager = TransportSecurityManager()
```

Comment:
I just noticed that this should probably have an `else` block that sets `transport_security_manager = None`: otherwise we'll pass the empty object down the stack.